### ISSUE TITLE
typo in DiMP_LTMU/pyMDNet/tracking/run_tracker.py

### ISSUE THREAD

**Issue by nihuo76** [Daikenan/LTMU#8]
line 8 in the following block:
`def forward_samples(model, image, samples, opts, out_layer='conv3'):`
`    model.eval()`
`    extractor = RegionExtractor(image, samples, opts)`
`    for i, regions in enumerate(extractor):`
`        if opts['use_gpu']:`
`            regions = regions.cuda()`
`        with torch.no_grad():`
`            feat = model(regions, out_layer=out_layer)`
`        if i==0:`
`            feats = feat.detach().clone()`
`        else:`
`            feats = torch.cat((feats, feat.detach().clone()), 0)`
`    return feats`

should the 'feat' in feat = model(regions, out_layer=out_layer) be 'feats' ?

**Comment by nihuo76** [comment]
Sorry I was wrong.
